svgdigitizer/svgplot.py

```diff
@@ -738,7 +718,7 @@ def curve(self):
         ... </svg>'''))
         >>> plot = SVGPlot(svg)
         >>> plot.curve
-        [(0.0, 100.0), (100.0, 0.0)]
+        Path(Line(start=0j, end=(1+1j)))
```

Comment:
This now returns the actual curve in the plot coordinate system, i.e., here the line going from (0, 0) to (1, 1).